### ISSUE TITLE
Feature Request: For the relation fields in Admin UI, add a "Open Record In New Tab" button

### ISSUE THREAD

**Issue by beltebelt** [pocketbase/pocketbase#4936]
This could just be a simple UI change on the Admin UI. Currently for the relation fields we have an "Info" control, hovering on which shows a tooltip with the related record details. We could have an additional control (or the same control can have both hover and click actions) to open the related record in a new tab. Just a button that does `window.open("https://<HOST>/_/#/collections?collectionId=<collectionId>&filter=&sort=-created&recordId=<relatedRecordId>","_blank")`

Something like:
![image](https://github.com/pocketbase/pocketbase/assets/155157292/fc25eba1-7552-4beb-8664-daa3441ea650)

I am assuming an anchor/button at [RecordInfo.svelte#L55](https://github.com/pocketbase/pocketbase/blob/1682db5c7215b6766c6e569833000f27932357e2/ui/src/components/records/RecordInfo.svelte#L55):

**Comment by beltebelt** [comment]
Proposed implementation:
```svelte
<!-- ui/src/components/records/RecordInfo.svelte -->
    {#if record && record.id && record.collectionId}
        <a
            href="/#/collections?collectionId={record.collectionId}&filter=&sort=-created&recordId={record.id}"
            target="_blank"
        >
            <i
                class="link-hint txt-sm ri-share-box-fill"
                use:tooltip={{
                    text: "Open Record In New Tab",
                    class: "code",
                    position: "top",
                }}
            />
        </a>
    {/if}
```
```svelte
<!-- ui/src/components/records/RecordsList.svelte -->
<!-- on:click={() => dispatch("select", record)} -->
on:click={(e) => {
    if (!e.target.classList.contains("ri-share-box-fill")) dispatch("select", record);
}}
```

**Comment by KISHORE-R-RAMESH** [comment]
Proposed implementation:


    {#if record && record.id && record.collectionId}
        <a
            href="/#/collections?collectionId={record.collectionId}&filter=&sort=-created&recordId={record.id}"
            target="_blank"
        >
            <i
                class="link-hint txt-sm ri-share-box-fill"
                use:tooltip={{
                    text: "Open Record In New Tab",
                    class: "code",
                    position: "top",
                }}
            />
        </a>
    {/if}


on:click={(e) => {
    if (!e.target.classList.contains("ri-share-box-fill")) dispatch("select", record);
}}

**Comment by KISHORE-R-RAMESH** [comment]
Proposed implementation:


    {#if record && record.id && record.collectionId}
        <a
            href="/#/collections?collectionId={record.collectionId}&filter=&sort=-created&recordId={record.id}"
            target="_blank"
        >
            <i
                class="link-hint txt-sm ri-share-box-fill"
                use:tooltip={{
                    text: "Open Record In New Tab",
                    class: "code",
                    position: "top",
                }}
            />
        </a>
    {/if}


on:click={(e) => {
    if (!e.target.classList.contains("ri-share-box-fill")) dispatch("select", record);
}}

**Comment by Salman9000** [comment]
+1
It is a hassle to even copy the id. Have to look through the collection

**Comment by ganigeorgiev** [comment]
Slightly different implementation is available as part of the [ongoing refactoring](https://github.com/pocketbase/pocketbase/discussions/4355) and will be included with the next v0.23.0 release soon (_there are no ETAs yet_).
